File: clients/java/android/src/main/java/com/wordnik/client/model/StatsHistory.java
package com.wordnik.client.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Date;
public class StatsHistory {
  @JsonProperty("date")
  private Date date = null;
  @JsonProperty("rootSymbol")
  private String rootSymbol = null;
  @JsonProperty("currency")
  private String currency = null;
  @JsonProperty("volume")
  private Double volume = null;
  @JsonProperty("turnover")
  private Double turnover = null;
  public Date getDate() {
    return date;
  }
  public void setDate(Date date) {
    this.date = date;
  }

  public String getRootSymbol() {
    return rootSymbol;
  }
  public void setRootSymbol(String rootSymbol) {
    this.rootSymbol = rootSymbol;
  }

  public String getCurrency() {
    return currency;
  }
  public void setCurrency(String currency) {
    this.currency = currency;
  }

  public Double getVolume() {
    return volume;
  }
  public void setVolume(Double volume) {
    this.volume = volume;
  }

  public Double getTurnover() {
    return turnover;
  }
  public void setTurnover(Double turnover) {
    this.turnover = turnover;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class StatsHistory {\n");
    sb.append("  date: ").append(date).append("\n");
    sb.append("  rootSymbol: ").append(rootSymbol).append("\n");
    sb.append("  currency: ").append(currency).append("\n");
    sb.append("  volume: ").append(volume).append("\n");
    sb.append("  turnover: ").append(turnover).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: clients/dart/lib/model/stats_history.dart
part of api;


@Entity()
class StatsHistory {
  
  @Property(name: 'date')
  DateTime date = null;
  

  @Property(name: 'rootSymbol')
  String rootSymbol = null;
  

  @Property(name: 'currency')
  String currency = null;
  

  @Property(name: 'volume')
  Number volume = null;
  

  @Property(name: 'turnover')
  Number turnover = null;
  
  StatsHistory();

  @override
  String toString()  {
    return 'StatsHistory[date=$date, rootSymbol=$rootSymbol, currency=$currency, volume=$volume, turnover=$turnover, ]';
  }

}

